### ISSUE TITLE
Updated preset for Google Nexus One

### ISSUE THREAD

**Issue by mdeslaur** [danielgtaylor/arista#31]
Since updating to Android 2.2 (Froyo), my Nexus One is more picky about the video files it will play.

This patch changes qtmux to mp4mux which solves the issues for me.

**Comment by mdeslaur** [comment]
From 93f162ec4164474206dae3b1a89e03f6a9fdcfa1 Mon Sep 17 00:00:00 2001
From: Marc Deslauriers marc.deslauriers@ubuntu.com
Date: Tue, 24 Aug 2010 08:09:10 -0400
Subject: [PATCH] Adjust Nexus One preset for 2.2 (Froyo) compatibility.

---

 presets/android.json |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/presets/android.json b/presets/android.json
index 1b5487a..b97d229 100644
--- a/presets/android.json
+++ b/presets/android.json
@@ -2,7 +2,7 @@
     "make": "Generic", 
     "model": "Android Phone", 
     "description": "Presets for all known Android phones", 
-    "version": "1.1", 
-    "version": "1.2", 
   "author": {
       "name": "Daniel G. Taylor", 
       "email": "dan@programmer-art.org"
  @@ -12,12 +12,12 @@
   "presets": [
       {
           "name": "Nexus One / Desire", 
-            "container": "qtmux", 
-            "container": "mp4mux", 
           "extension": "mp4", 
           "icon": "file://android-nexus-one.svg", 
           "vcodec": {
               "name": "x264enc", 
-                "container": "qtmux", 
-                "container": "mp4mux", 
               "width": [
                   320, 
                   800
  @@ -36,7 +36,7 @@
           }, 
           "acodec": {
               "name": "faac", 
-                "container": "qtmux", 
-                "container": "mp4mux", 
               "width": [
                   8, 
                   24
  -- 
  1.7.1

**Comment by danielgtaylor** [comment]
Nice find. I've talked to the GStreamer guys and you are indeed right that we should use mp4mux instead of qtmux. I've updated the presets (and updated a few others) and pushed it live.

See 2d5a3e601f0a4401fa22883e36fc4ea031e1f285 and danielgtaylor/arista-website@e399d2e6a2ab8d469134b3402f5b486e0fa5c7d3

Users should get the updated presets the next time they run Arista or something tonight or tomorrow.

Thanks for the patch!
